charts/bitcoind/templates/statefulset.yaml

```diff
@@ -111,6 +111,47 @@ spec:
             successThreshold: 3
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+        {{- if and .Values.descriptor.secretName .Values.descriptor.secretKey }}
```

Comment:
Does this evaluate to true when they are set to `""`?

Reply:
no, it doesn't.
Confirmed by having no `descriptor-import` container for `bitcoind-0` in dev.